src/main/scala/cognite/spark/v1/StringDataPointsRelation.scala

```diff
@@ -86,14 +86,16 @@ class StringDataPointsRelationV1(config: RelationConfig)(override val sqlContext
       (
         StringDataPointsFilter(Some(id), None),
         client.dataPoints
-          .queryStringsById(id, lowerTimeLimit, upperTimeLimit, config.limit))
+          .queryStringsById(id, lowerTimeLimit, upperTimeLimit, config.limit)
```

Comment:
Might be nicer with a for comprehension?

Reply:
Couldn't find a nicer one, unless you meant the `ids.map{ id` part, but that's pretty much the same. Gets tricky with `IO`, do you have a suggestion?

Reply:
Not really nicer, you can keep it as is.
```
val itemsIOFromId = for {
      id <- ids
    } yield (
      StringDataPointsFilter(Some(id), None),
      for {
        queryResults1 <- client.dataPoints.queryStringsById(id, lowerTimeLimit, upperTimeLimit, config.limit)
      } yield queryResults1.flatMap(_.datapoints)
    )
```